src/public/js/route/userRoute.js

```diff
@@ -1,21 +1,26 @@
-async function userRoute(el, logoutTemplate, loginTemplate, api, showError) {
-    //check if logged in or not
-    const user = getUser();
-    let html;
-
-    if(user) {
-        html = logoutTemplate({user: getUser()});
-    } else {
-        html = loginTemplate();
+async function userRoute(el, logoutTemplate, api, showError) {
+    async function logoutFunction() {
```

Comment:
please move the inner function out of the outer function. This code looks really confusing.